configuration/components/tracing.libsonnet

```diff
@@ -92,7 +92,8 @@ function(params) {
         labels: newCommonLabels(component),
       },
       spec: {
-        strategy: 'allinone',
+        strategy: tracing.config.jaeger.strategy,
```

Comment:
wait that does not make sense, because its the default configuration?

Reply:
We should establish some convention here. This PR essentially forces us to change `tracing.libsonnet` every time we need to make a new field configurable in the Jaeger CR. A more flexible solution might be better option. For instance a consumer of the `tracing.libsonnet` could define full Jaeger CR.

Reply:
Meaning that we store an `JaegerSpec` with default values somewhere like `components/jaeger.libsonnet`, which could be adapted in main and passed via the tracing configuration?